cfme/common/provider.py

```diff
@@ -672,6 +672,19 @@ def load_all_provider_vms(self):
             sel.click(details_page.infoblock.element("Relationships", self.vm_name))
             return True
 
+    def load_all_provider_nodes(self):
+        """ Loads the list of nodes that are running under the provider.
+
+        If it could click through the link in infoblock, returns ``True``. If it sees that the
+        number of instances is 0, it returns ``False``.
+        """
+        self.load_details()
+        if details_page.infoblock.text("Relationships", "Nodes") == "0":
```

Comment:
this would look better like bool(details_page.infoblock.text("Relationships", "Nodes"))